templates/plugin/abrt.yml.erb

```diff
@@ -0,0 +1,3 @@
+---
+# Abrt integration
```

Comment:
You should copy https://github.com/theforeman/smart_proxy_abrt/blob/master/settings.d/abrt.yml.example as a starting point for this template.